packages/common/src/chains.ts

```diff
@@ -448,4 +448,122 @@ export const chains: ChainsDict = {
       'enrtree://AKA3AM6LPBYEUDMVNU3BSVQJ5AD45Y7YPOHJLEF6W26QOE4VTUDPE@all.sepolia.ethdisco.net',
     ],
   },
+  holesky: {
+    name: 'holesky',
+    chainId: 17000,
+    networkId: 17000,
+    defaultHardfork: 'paris',
+    consensus: {
+      type: 'pos',
+      algorithm: 'casper',
+    },
+    comment: 'PoS test network to replace Goerli',
+    url: 'https://github.com/eth-clients/holesky/',
+    genesis: {
+      baseFeePerGas: '0x3B9ACA00',
+      difficulty: '0x01',
+      extraData: '0x686f77206d7563682069732074686520666973683f',
+      gasLimit: '0x17D7840',
+      nonce: '0x0000000000001234',
+      timestamp: '0x65046234',
+    },
+    hardforks: [
+      {
+        name: 'chainstart',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'homestead',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'tangerineWhistle',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'spuriousDragon',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'byzantium',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'constantinople',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'petersburg',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'istanbul',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'muirGlacier',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'berlin',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'london',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        // The forkHash will remain same as mergeForkIdTransition is post merge,
+        // terminal block: https://sepolia.etherscan.io/block/1450408
```

Comment:
Is this comment (or at least this link) still applicable?

Reply:
No this is not applicable in this case, thanks :smile: